src/app/containers/main/index.jsx

```diff
@@ -65,12 +67,20 @@ class Main extends Component {
     }
 
     searchRequest(searchTerm, offset) {
+        this.setState(initialState);
+
         if (searchTerm) {
+            const error = {
+                message: `Searching for "${searchTerm}"...`,
+                imageUrl: loadingImage,
+            };
+            this.setState({error});
```

Comment:
I do realize using `error` here is a bit weird, but it's just so available and ready to be used.